pocketoptionapi_async/websocket_client.py

```
@@ -124,176 +127,172 @@ def __init__(self):
             message: Message to send
         """
         if not self.websocket or self.websocket.closed:
             raise WebSocketError("WebSocket is not connected")
-            
+
         try:
             await self.websocket.send(message)
             logger.debug(f"Sent message: {message}")
         except Exception as e:
             logger.error(f"Failed to send message: {e}")
             raise WebSocketError(f"Failed to send message: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)